### ISSUE TITLE
Unable to override CSS without !important

### ISSUE THREAD

**Issue by smlombardi** [ng-bootstrap/ng-bootstrap#2090]
### Version of Angular, ng-bootstrap, and Bootstrap:

Angular: 5.1.3
ng-bootstrap: 1.0b9
Bootstrap: 4.0b3

I am trying to style the date pickers to look like we styled the older date pickers in Bootstrap 3.

For example, the weekdays in your code are italic and teal.  I want them to be roman and black.

In the code, they are output as
```
<div _ngcontent-c8="" class="ngb-dp-weekday small text-center font-italic">
     Sa
</div>
```

So, I try
```
.ngb-dp-weekday {
  font-style: normal;
  color: $gray-500;
}
```

Nothing happens.  Inspecting the code in Dev Tools shows that my code is in `styles_bundle.css`, but your code is in `<style></style>`.  since your code is on the page itself, it overrides the imported css.

If I use `important`, **and** all the classes, I can override, but I'd rather not:

```
.ngb-dp-weekday.small.text-center.font-italic {
  font-style: normal !important;
  color: $gray-500 !important;
}
```

It's worth adding this happens primarily when using `--extract-css`, which we need to get sourcemaps.  If I disable the flag, I can override the color without `!important`, although you still add the class `font-italic`, which is set to `!important` in your (or Bootstrap's) css, and trumps mine.

**Comment by bastienmoulia** [comment]
Indeed, ng-bootstrap should not use the bootstrap class containing properties with`!important` because it can't be override (like [font-italic](https://github.com/twbs/bootstrap/blob/b72bd051ab605c5de9ed21764d8e696d3aba2e5c/scss/utilities/_text.scss#L38)).

Bootstrap class should only be used only for sizing and theme color.

**Comment by pkozlowski-opensource** [comment]
This have been fixed in #2296 already
